internal/utils/utils.go

```diff
@@ -78,3 +79,34 @@ func CapitalizeFirst(s string) string {
 	r[0] = unicode.ToUpper(r[0])
 	return string(r)
 }
+
+func FileExists(file string) bool {
+	if absPath, err := filepath.Abs(file); err == nil {
+		file = absPath
+	}
+
+	info, err := os.Stat(file)
+	if os.IsNotExist(err) {
+		return false
+	}
+
+	return !info.IsDir()
+}
+
+func SanitizeFilePath(path string) string {
+	sanitizedPath := path
+	if strings.HasPrefix(path, "~/") {
```

Comment:
Surprised none of the filpath libs can account for this

Reply:
filepath doesn't cover it as far as my research has found, same issue we saw in sdk-gen-config